src/components/Tooltip/index.js

```diff
@@ -20,7 +20,12 @@ const Tooltip = props => {
     }
   })();
 
-  return <tooltip bgColor={bgColor} color={color} text={text} />;
+  return (
+    <TooltipWrapper>
+      {props.children}
```

Comment:
hey, what does this line do? @AaronJThompson

Reply:
Which line are you referring to?